client/src/layouts/CommunitiesLayout.js

```diff
@@ -0,0 +1,17 @@
+import { Route, Switch } from "react-router-dom";
+import AppHeader from "../components/app/AppHeader";
+import MyCommunitiesPage from "../pages/communities/MyCommunitiesPage";
+import "./CommunitiesLayout.css";
+
+export default function CommunitiesLayout() {
+  return (
+    <>
+      <AppHeader className="CommunitiesHeader" />
```

Comment:
I think you are rerendering inside each router. Why don't you keep it outisde the switch?

Reply:
I want to provide different header bar actions depending on the current layout  / location but keep the header style / layout and common icons / actions etc. (e.g. the plus-button in the header bar (top right) shall open the "create new community" form when inside the MyCommunities-Layout - and the "create new community user group" when inside the CurrentCommunity-Layout etc.)